editor/components/TableDateSelector.tsx

```diff
@@ -117,6 +117,8 @@ export default function TableDateSelector({
             className="form-control"
             onChange={(date) => {
               if (date) {
+                // set the hours to the end of the date selected
+                date.setHours(23, 59, 59, 999);
```

Comment:
Do we know why the current time isn't being used in the picker for the start of the range too? I was looking through the code, and I'm probably just missing something!

Reply:
I dont know and it confused me because I thought I was going to have to move it to start of the day. 
This [line](https://github.com/cityofaustin/vision-zero/blob/main/editor/utils/dates.ts#L25) sets the target start date to midnight but i dont think that date affects the date picker at all.